core/src/test/java/org/testcontainers/containers/ExposedHostTest.java

```diff
@@ -30,31 +33,59 @@ public static void setUpClass() throws Exception {
 
         server.start();
         Testcontainers.exposeHostPorts(server.getAddress().getPort());
+        
+        serverPortMapping = HttpServer.create(new InetSocketAddress(0), 0);
```

Comment:
Sorry we're on it another time, but:

I think that's enough to use the same server, and expose the same port two more times with:
```java
Testcontainers.exposeHostPorts(ImmutableMap.of(
        serverPortMapping.getAddress().getPort(), 80,
        serverPortMapping.getAddress().getPort(), 81
);
```

and then add one more test where you check that both port 80 and 81 are assigned.
Why two? Because we need to test the composite Map key in a form of an entry you added.

Reply:
next round. ding ding ding ;-)

Unfortunatelly it is not possible with one `ImmutableMap` statement due to `IllegalArgumentException: Multiple entries with same key`

Reply:
I think I just broke it 😂 That's a good catch, I will revert

Reply:
Done, waiting for CI